tools/go-agent/config/agent.default.yaml

```diff
@@ -97,3 +97,5 @@ plugin:
     redis:
       # Limit the bytes size of redis args request
       max_args_bytes: ${SW_AGENT_PLUGIN_CONFIG_REDIS_MAX_ARGS_BYTES:1024}
+    gin: 
+      collect_http_headers: ${SW_AGENT_PLUGIN_CONFIG_GIN_COLLECT_HTTP_HEADERS:}
```

Comment:
I think you missing the configuration declaration. please take a look at other plugins, such as https://github.com/apache/skywalking-go/blob/main/plugins/go-redisv9/config.go

Reply:
And the default value should be `false` I think.

Reply:
Yes, config.go is missing